student.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'person'
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(classroom, age, _name = 'Unknown', _parent_permission: true)
+    super(age, _name = 'Unknown', _parent_permission: true)
+    @classroom = classroom
+  end
```

Comment:
- [ ] Your `super` method prevents the passed-in parameters from being used in the object creation. After passing name and parent_permission arguments, your super method reverts everything to "Unknown", and `true` respectively.

- [ ] The underscore for does not seem to work well with the `_parent_permission` part. You probably need to refactor to something like so:
  ```
  def initialize(classroom, age, _name = 'Unknown', parent_permission: true)
    super(age, _name, parent_permission: parent_permission)
    @classroom = classroom
  end
  ```